fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -47,8 +43,22 @@
 # TODO: need to account for spelling variations
 # TODO: need to account for overlap. CDA is a subset of C-CDA, so need to prune results for these miscatches.
 KEYWORDS = {
-    'code_systems': ['DICOM', 'SNOMED', 'LOINC', 'ICD', 'NDC', 'RxNorm'],
-    'product_families': ['V2', 'Version 2', 'CDA', 'C-CDA', 'V3', 'Version 3'],
```

Comment:
I laid these lists out vertically for consistency. For the other lists, it was just easier for me to do that since I was copy/pasting from a separate text editor. I'm not sure if there's an official Python style recommendation for lists (vertical or horizontal layout). I'm open to (a) keeping this way, (b) moving this into a separate file (could be JSON) to reduce number of lines in this file, or (c) just changing them all to horizontal.